extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -134,3 +134,16 @@ export function doubleclickIsA4AEnabled(win, element) {
   }
   return enableA4A;
 }
+
+export const adConfig = {
```

Comment:
I tried removing them all one by one and no tests broke. @keithwrightbos do you know which ones we definitely don't need here?